src/libraries/System.Runtime.InteropServices.RuntimeInformation/src/System.Runtime.InteropServices.RuntimeInformation.csproj

```diff
@@ -45,5 +46,6 @@
     <Reference Include="System.Runtime" />
     <Reference Include="System.Runtime.InteropServices" />
     <Reference Include="System.Threading" />
+    <Reference Include="System.Memory" />
```

Comment:
I am wondering why we need this

Reply:
this was needed to turn a string into a `ReadOnlySpan<char>` by using `string.AsSpan`